### PR TITLE
Add CSV for 0.6.1

### PR DESCRIPTION
**Pull request by LaVLaS**
Adding the CSV for the 0.6.1 release. For the example kfdef, I used the odh-manifest [kfctl_openshift.yaml](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift.yaml)

### CODE REVIEW

File: deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    Open Data Hub is a meta-project that integrates open source projects into a practical solution. It aims to foster collaboration between communities, vendors, user-enterprises, and academics following open source best practices. The open source community can experiment and develop intelligent applications without incurring high costs and having to master the complexity of modern machine learning and artificial intelligence software stacks.
+
+    ### Components
+    * JupyterHub v3.0.7 - Open source multi-user notebook platform w/ GPU support
+    * Apache Spark v2.4.5 - Unified analytics engine for large-scale data processing
+    * Prometheus v2.7.1 - Monitoring and alerting tool
+    * Grafana v6.2.4 - Data visualization and monitoring
+    * Airflow v1.10.2 - Workflow management
+    * AI-Library v1.0 - An open source collection of machine learning algorithms
+    * Seldon v1.1.0 - Open source platform for deploying machine learning models
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml:259]: AI-Library should 0.6.0

File: deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+              }
+            ],
+            "repos": [
+              {
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v0.7-branch-openshift"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/master"
```
Comment (vpavlin) [inline deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml:219]: We should create a release in `odh-manifests` and point the example to it (at least as long as we keep manifests and operator versions in sync)

File: deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+              name: opendatahub-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: opendatahub-operator
+            spec:
+              containers:
+              - command:
+                - kfctl
```
Comment (vpavlin) [inline deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml:312]: How was the image built? Does it have kfctl or opendatahub-operator binary?

Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml:312]: Good catch. Tested and updated to run `opendatahub-operator`

File: deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+                "name": "kf-manifests",
+                "uri": "https://github.com/opendatahub-io/manifests/tarball/v0.7-branch-openshift"
+              },
+              {
+                "name": "manifests",
+                "uri": "https://github.com/opendatahub-io/odh-manifests/tarball/master"
+              }
+            ],
+            "version": "v0.7-branch-openshift"
+          },
```
Comment (nakfour) [inline deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml:223]: Change to match the manifest version 0.6.1

**Comment by nakfour** [comment]
@nakfour @vpavlin re-review

File: deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml (hunk 2 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml]: updated to pull the tarball for `v0.6.1`

File: deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml (hunk 4 above)
Comment (LaVLaS) [reply deploy/olm-catalog/opendatahub/0.6.1/opendatahub-operator.v0.6.1.clusterserviceversion.yaml:223]: Updated

**Comment by vpavlin** [comment]
LGTM

**Pull request merged**